sjsonnet/src-js/sjsonnet/SjsonnetMain.scala

```diff
@@ -6,7 +6,7 @@ import scala.scalajs.js.annotation.{JSExport, JSExportTopLevel}
 
 @JSExportTopLevel("SjsonnetMain")
 object SjsonnetMain {
-  def createParseCache() = collection.mutable.HashMap[(Path, String), Either[String, (Expr, FileScope)]]()
+  def createParseCache() = new DefaultParseCache
```

Comment:
Can we deprecate this method? We have the default encapsulated nicely in `new DefaultParseCache` now and it is used directly in several places, so this method seems superfluous.

Reply:
Yes, that makes sense. Updated